seo_app/content.py

```diff
@@ -0,0 +1,40 @@
+# import urllib.request
+# # import urlllib2
+# # try:
+# #     from urllib.request import Request, urlopen  # Python 3
+# # except ImportError:
+# #     from urllib2 import Request, urlopen  # Python 2
+# myfile = None
+# link = "http://www.somesite.com/details.pl?urn=2344"
+# with urllib.request.urlopen (link) as url:
+        
+#         myfile = url.read()
+# print(myfile)
+
+
+from selenium import webdriver
+from bs4 import BeautifulSoup
+import pandas as pd
+from webdriver_manager.chrome import ChromeDriverManager
+
+import ipdb;ipdb.set_trace()
+
+driver = webdriver.Chrome(ChromeDriverManager().install())
+
+products=[] #List to store name of the product
+prices=[] #List to store price of the product
+ratings=[] #List to store rating of the product
+driver.get("<a href=https://www.flipkart.com/laptops/>https://www.flipkart.com/laptops/</a>~buyback-guarantee-on-laptops-/pr?sid=6bo%2Cb5g&amp;amp;amp;amp;amp;amp;amp;amp;amp;uniq")
```

Comment:
This project is not for a specific URL based scraping, but for generic scraping of webpages.